internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -175,9 +177,13 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	// TODO: need to be refactored because we can error out earlier.
```

Comment:
nit: can you add the package where the refactor should go to? it sounds like it can go to `manifest` validation 💭